### PR TITLE
Meta lamp/lending pool/frontend/abstract contract polling

### PR DESCRIPTION
**Pull request by performanceArtist**

### CODE REVIEW

**Comment by stanislav-az** [comment]
It would be helpful if we could use an "uninstall" script to remove all frontend generated stuff. Something like `rm -rf node_modules/ generated/ output/ plutus-purs/ .spago/ `, but with full paths for safety.

**Comment by stanislav-az** [comment]
We need to also show the debt of a user from `UserConfig.ucDebt`, it is very helpful while debugging

**Comment by stanislav-az** [comment]
Pool funds state update is not showed on `repay` & `borrow`endpoints for some reason

**Comment by stanislav-az** [comment]
`TxnValidationFail` does not stop polling and results in `TooManyRetries` error

**Comment by performanceArtist** [comment]
> It would be helpful if we could use an "uninstall" script to remove all frontend generated stuff. Something like `rm -rf node_modules/ generated/ output/ plutus-purs/ .spago/ `, but with full paths for safety.

Hardly a common practice. Why would you want to remove all of it at once anyway?

**Comment by stanislav-az** [comment]
> Hardly a common practice

I don't know, for me `npm` fails sometimes and the only way to fix it is to delete some of these folders.

File: MetaLamp/lending-pool/client/src/Capability/PollContract.purs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  | PollResponseError String
+
+derive instance genericPollError :: Generic PollError _
+
+instance showPollError :: Show PollError where
+  show = genericShow
+
+data LeftPoll
+  = Continue
+  | ResponseError String
```
Comment (stanislav-az) [inline MetaLamp/lending-pool/client/src/Capability/PollContract.purs:34]: What is the difference between `PollError` and `LeftPoll`? They have two similar constructors - `PollResponseError` and `ResponseError`, can we merge these types?

**Pull request merged**